redisinsight/ui/src/components/side-panels/panels/enablement-area/EnablementArea/components/DeleteTutorialButton/DeleteTutorialButton.tsx

```diff
@@ -20,7 +22,10 @@ const DeleteTutorialButton = (props: Props) => {
   const { id, label, onDelete, isLoading } = props
   const [isPopoverDeleteOpen, setIsPopoverDeleteOpen] = useState<boolean>(false)
 
-  const handleClickDelete = () => {
+  const handleClickDelete = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
```

Comment:
Minor enhancement to prevent expland/collapse of the section behind when clicking the delete button.

https://github.com/user-attachments/assets/2efd0611-53fe-403c-a335-800e34c3860d